kolibri/plugins/management/kolibri_plugin.py

```diff
@@ -26,6 +26,6 @@ class ManagementInclusionHook(hooks.ManagementSyncHook):
     bundle_class = ManagementAsset
 
 
-class ManagementNavItem(UserNavigationHook):
-    label = _("Management!")
-    url = '#'
+class ManagementNavItem(NavigationHook):
```

Comment:
Also added manage, just so that I could get a few more links to modules during development. Might not be appropriate. Again, input is welcome.